docs/source/release_notes.rst

```diff
@@ -11,7 +11,8 @@ Future Release
     * Documentation Changes
     * Testing Changes
         * Use codecov action v3 (:pr:`1422`)
-	* Add workflow to kickoff EvalML unit tests on commit to main (:pr:`1424`, :pr:`1426`)
+	    * Add workflow to kickoff EvalML unit tests on commit to main (:pr:`1424`, :pr:`1426`)
```

Comment:
Extra indentation here should be removed

Reply:
fixed